uploadFile.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
```

Comment:
why no functions in this file? each check could be a function

Reply:
functions and tests done